### DISCUSSION TITLE
Gupshup Integration - Inactive Channel

### DISCUSSION THREAD

**Discussion by thundersparkf** [chatwoot/chatwoot#2517] [Q&A] [answered]
I was experimenting with the Gupshup integration in the self hosted version and I am facing this issue. I'm a Ruby novice that's why I hadn't actually pushed any commits about this in the existing issue. 

<img width="292" alt="Screenshot 2021-06-26 at 12 47 59 AM" src="https://user-images.githubusercontent.com/64605024/123474815-28fa8d00-d618-11eb-93fc-5ffce0a3565a.png">

### What issues cause channels to not be active when you click on them?

**Comment by sojan-official** [comment, marked as answer]
@thundersparkf  ref: https://github.com/chatwoot/chatwoot/blob/b705eb39e2a89ad8d2802a4ede6c90fcfcd18f12/app/javascript/dashboard/components/widgets/ChannelItem.vue#L75

Add your channel name to the array so that chatwoot can identify it as active. 

Also, feel free to use our discord if you have any questions, while you are looking to contribute: https://discord.gg/cJXdrwS
